scripts/apothecary/formulas/cairo.sh

```diff
@@ -30,40 +30,81 @@ GIT_TAG=$VER
 # download the source code and unpack it into LIB_NAME
 function download() {
 	curl -LO http://cairographics.org/releases/cairo-$VER.tar.xz
-	tar -xf cairo-$VER.tar.xz
-	mv cairo-$VER cairo
-	rm cairo-$VER.tar.xz
-
+	if [ "$TYPE" == "vs" ] ; then
+		7z e cairo-$VER.tar.xz
```

Comment:
right I couldn't get the tar that is packaged with git bash to unpack the xz files and had 7zip on my computer since its my goto for compression/decompression. Hopefully that wont be a problem in the future